src/IBM.Watson.TextToSpeech.v1/Websockets/AWebSocketClient.cs

```diff
@@ -129,28 +130,33 @@ protected async Task HandleResults()
                 }
 
                 var message = Encoding.UTF8.GetString(buffer, 0, count);
-
-                if (message.Contains(BinaryStreams))
-                {
-                    var json = JObject.Parse(message);
-                    string contentType = json[BinaryStreams][0][ContentType].ToString();
-                    OnContentType(contentType);
-                }
-                else if (message.Contains(Marks))
+                try
                 {
                     var json = JObject.Parse(message);
-                    JToken marks = json[Marks];
-                    MarkTiming markTiming = json.ToObject<MarkTiming>();
-                    OnMarks(markTiming);
-                }
-                else if (message.Contains(Words))
-                {
-                    var json = JObject.Parse(message);
-                    JToken words = json[Words];
-                    WordTiming wordTimings = json.ToObject<WordTiming>();
-                    onTimings(wordTimings);
+                    if (json.ContainsKey(Error))
+                    {
+                        OnError(new Exception(json[Error].ToString()));
```

Comment:
Is this where the actual error is? can you post what we can expect to see here

Reply:
it is received like:
```
{
  "error": "Tag <prosody> is missing required attribute->pitch, rate, or volume"
}
```
and sent as an exception like `System.Exception: Tag <prosody> is missing required attribute->pitch, rate, or volume`